sbag/generators/javascript/templates/app/src/app/modules/__entity_name__/__entity_name__-routing.module.ts.jinja

```diff
@@ -0,0 +1,19 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+import { {{ entity.name }}Component } from './{{ entity.name | lower }}/{{ entity.name | lower }}.component';
+import { {{ entity.name }}ListComponent } from './{{ entity.name | lower }}-list/{{ entity.name | lower }}-list.component';
+import { {{ entity.name }}FormComponent } from './{{ entity.name | lower }}-form/{{ entity.name | lower }}-form.component';
+
+const routes: Routes = [
+  { path: ':id', component: {{ entity.name }}Component },
+  { path: '', component: {{ entity.name }}ListComponent },
+  { path: ':id/update', component: {{ entity.name }}FormComponent }
```

Comment:
@vtanja @veca22 @nikolina-sarenac What could be a good name for FormComponent route?

Reply:
Sholudn't we specify here separate routes, for example:
 { path: 'add', component: {{ entity.name }}FormComponent } 
 { path: ':id/update', component: {{ entity.name }}FormComponent } ?

Reply:
You are rightm fix comming